.github/ISSUE_TEMPLATE/icon_request.yml

```diff
@@ -0,0 +1,43 @@
+name: Icon request
+description: Request a new icon for Simple Icons
+labels: [new icon]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        We won't add non-brand icons or anything related to illegal services. If in
+        doubt, open an issue and we'll have a look. Before opening a new issue please
+        search for duplicate or closed issues or PRs. If you find one for the brand
+        your requesting then leave a comment on it or add a reaction to the original
+        post.
+
+  - type: input
+    attributes:
+      label: Brand Name
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Website
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Alexa Rank
+      description: >-
+        The Alexa rank can be retrieved at https://www.alexa.com/siteinfo/. Please see
+        our contributing guidelines for more details on how we assess a brand's
+        popularity.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Official resources for icon and color
```

Comment:
Not sure if the official resources for icon and color should be required or not

Reply:
I think we should make it required and add in the description that you can set "none" if you couldn't find any, hopefully this encourage reporters to do some research themselves - at the very least it prevents people from just submitting the form without seeing this question.